src/components/modules/ui.ts

```diff
@@ -390,13 +390,23 @@ export default class UI extends Module {
   private backspacePressed(event: KeyboardEvent): void {
     const {BlockManager, BlockSelection, Caret} = this.Editor;
 
+    /** Prevent block removal while editing an input inside settings */
+    if (this.Editor.BlockSettings.opened) {
+      if (document.activeElement.tagName === 'INPUT') {
```

Comment:
we have a useful method for that, see `dom.canSetCaret()`

Reply:
Ok, thanks. It has more features. I'll update the PR this evening of I have time.